specification/keyvault/Security.KeyVault.Keys/client.tsp

```diff
@@ -120,3 +120,38 @@ using KeyVault;
   "RsaAesKeyWrap384",
   "javascript"
 );
+
+// Rust configuration
+@@clientName(KeyCreateParameters, "CreateKeyParameters", "rust");
+@@clientName(KeyExportParameters, "ExportKeyParameters", "rust");
+@@clientName(KeyImportParameters, "ImportKeyParameters", "rust");
+@@clientName(KeyReleaseParameters, "ReleaseParameters", "rust");
+@@clientName(KeyRestoreParameters, "RestoreKeyParameters", "rust");
+@@clientName(KeySignParameters, "SignParameters", "rust");
```

Comment:
Sign vs SignKey?

Reply:
Yes. We don't repeat "key". All these changes I already meticulously verified against all our other languages' APIViews.